api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -224,6 +224,16 @@ message GrpcJsonTranscoder {
 
   // If true, try to match the custom verb even if it is unregistered. By
   // default, only match when it is registered.
+  //
+  // According to the http template syntax(https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L226-L231),
+  // the custom verb is `":" LITERAL` at the end of http template. Only when a
+  // http template has a custom verb defined, it can can only match a request
```

Comment:
We need to list out the behavior change, not need to mention if it is the spec compliance or not. which is not important.
how about:
```
for a request with /foo/bar:baz,   if ":baz" is not registered in any url_template, here is the behavrios change
* if the field is not set, ":baz" will not be treated as custom verb, so it will match /foo/{x=*}.  
* if the field is set, ":baz" is treated as custom verb,  so it will NOT match /foo{x=*} since the template dosn't use any custom verb.
```